tests/tests_pytorch/serve/test_servable_module_validator.py

```diff
@@ -41,6 +43,7 @@ def test_servable_module_validator_with_trainer():
         limit_train_batches=2,
         limit_val_batches=0,
         callbacks=[callback],
+        strategy="ddp_spawn",
```

Comment:
should we test thus with all the supported strategies just to be sure?

Reply:
I think it is an overkill right now.